frontend/src/pages/frontPage/Panel.jsx

```diff
@@ -1,13 +1,15 @@
 import Nav from "../../components/layout/Nav";
 import { Outlet } from "react-router-dom";
 
+import SignUp from "../../components/layout/SignUp";
 const Panel = (props) => {
-    const {isLoggedIn } = props;
-    return(
-        <>
-            <Nav isLoggedIn={isLoggedIn}/>
-            <Outlet/>
-        </>
-    )
-}
-export default Panel;
\ No newline at end of file
+  const { isLoggedIn } = props;
+  return (
+    <>
+      <Nav isLoggedIn={isLoggedIn} />
+      <SignUp isLoggedIn={isLoggedIn} />
```

Comment:
This is going to be really annoying to have over the screen all the time when it gets merged in. Can we find a way to show this only when someone clicks on the "register" nav link?

You could:
- Control whether this renders using a state variable like `showingSignupForm` that flips true/false when the nav link is clicked.
- Register the signup form at the `/register` route, which is currently when I am navigated when I click the register button (and that's an empty page right now since we don't have anything there).

I would lean towards the first since we've designed this to just be a popup in front of whatever else is on the screen. In that case, I would probably move this component into the `Nav` component where that state would live.